MapboxCoreNavigation/Dictionary.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+
+extension Dictionary where Key == Int, Value: MGLStyleValue<NSNumber> {
+    /**
+     Returns a copy of the stop dictionary with each value multiplied by the given factor.
+     */
+    public func multiplied(by factor: Double) -> Dictionary {
+        var newCameraStop: [Int: MGLStyleValue<NSNumber>] = [:]
+        for stop in MBRouteLineWidthByZoomLevel {
```

Comment:
Because this line refers to `MBRouteLineWidthByZoomLevel` instead of `self`, this code:

```swift
let stops = [5: MGLStyleValue(rawValue: 1)]
stops.multiplied(by: 1)
```

magically produces a dictionary equivalent to `MBRouteLineWidthByZoomLevel`, even though `MBRouteLineWidthByZoomLevel` was never used explicitly.